mu-plugins/10up-plugin/src/Assets.php

```diff
@@ -0,0 +1,273 @@
+<?php
+/**
+ * Assets module.
+ *
+ * @package TenUpPlugin
+ */
+
+namespace TenUpPlugin;
+
+use TenupFramework\Module;
+use TenupFramework\ModuleInterface;
+
+/**
+ * Assets module.
+ *
+ * @package TenUpPlugin
+ */
+class Assets implements ModuleInterface {
+
+	use Module;
+
+	/**
+	 * Can this module be registered?
+	 *
+	 * @return bool
+	 */
+	public function can_register() {
```

Comment:
Could this be the default?

Reply:
That change would need to be made in the framework, but I don't think it's a bad idea 🤔